Automattic/wp-calypso
# State: Cleanup unused gutenbergOptInOut state

#### Changes proposed in this Pull Request

* State: Cleanup unused gutenbergOptInOut state

#### Testing instructions

* Verify the removed code isn't used anywhere.
* Verify the editor loads well and works like before for a simple site, Atomic site, and a Jetpack site.
* Verify all tests still pass.
